test/rules/no-unsafe-any/unknown/test.ts.lint

```diff
@@ -0,0 +1,82 @@
+[typescript]: >= 3.0.0
+
+import importEquals = require("./commonjsModule");
+import importAlias = importEquals;
+namespace N { export const x: any = 0; }
+import importQualifiedName = N.x;
+import * as namespaceImport from "./es6Module";
+
+declare function takesUnknown(a: any, ...bs: any[]): void;
+takesUnknown(x, x);
```

Comment:
don't you need to declare `x` in this scope? what is the inferred type of `x` here?